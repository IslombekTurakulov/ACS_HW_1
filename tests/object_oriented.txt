object_oriented 1925 35 single
object_oriented 1999 65 multiple
object_oriented 2006 54 multiple
object_oriented 2019 96 single
object_oriented 2014 24 interface
object_oriented 2005 12 single