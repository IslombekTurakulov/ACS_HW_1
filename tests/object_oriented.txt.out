Sorted container:
Container contains 6 elements.
0: Object Oriented: name = dfgdfg popularity = 35%, age = 1925, legacy = 0
1: Object Oriented: name = qwerty popularity = 12%, age = 2005, legacy = 0
2: Object Oriented: name = werwer popularity = 54%, age = 2006, legacy = 1
3: Object Oriented: name = qweqw popularity = 65%, age = 1999, legacy = 1
4: Object Oriented: name = cxvxc popularity = 96%, age = 2019, legacy = 0
5: Object Oriented: name = zxc popularity = 24%, age = 2014, legacy = 2
